yoda_dbt2looker/generator.py

```diff
@@ -534,16 +534,20 @@ def lookml_view_from_dbt_exposure(model: models.DbtModel, dbt_project_name: str)
 #     return ref_str
 
 
-def _convert_all_refs_to_relation_name(ref_str: str) -> str:
+def _convert_all_refs_to_relation_name(
+    ref_str: str, adjust_whitespaces: bool = True
+) -> str:
     reg_ref = r"ref\(\s*\'(\w*)\'\s*\)"
     matches = re.findall(reg_ref, ref_str)
     if not matches or len(matches) == 0:
         return ref_str
 
-    ref_str = ref_str.replace(" ", "")
+    if adjust_whitespaces:
```

Comment:
two questions 
1. since you are replacing all spaces in ref_str it isn't replacing just whitespaces so I would change the naming here , if you want to handle only whitespace use `ref_str.strip()` .
2. why you are checking the same condition twice can't it be union ?